.github/workflows/build-test-analyze.yml

```diff
@@ -69,6 +69,71 @@ jobs:
           flags: unittests
           name: codecov-coverage
           token: ${{ secrets.CODECOV_TOKEN }}
+  test:
+    name: Instrumented tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
I think we can fail fast here, `connectedCheck` is a relatively long lasting task 😅

Reply:
Good point; I had it in there to do some testing but we should enable it from now on 👍